pvm.py

```diff
@@ -12,10 +12,12 @@ def _init_logger():
 	logger.setLevel(logging.INFO)
 	handler = logging.StreamHandler(sys.stderr)
 	# Check if the `log` directory exists, create one if not.
-	log_path = "./log/{:%Y-%m-%d %H:%M:%S}.log"
+	log_path = "/home/pi/PVM/log/{:%Y-%m-%d %H:%M:%S}.log"
```

Comment:
maybe this is only a quick fix, and not the actual fix.
should we leave it with an absolute path like this?

suggestion: retrieve the absolute path by retrieving where `pvm.py` file being executed is located?

Reply:
Yes, this is a quick fix in the log path. We should fix all relative paths in #8.
Because all absolute paths are related to this script.

Reply:
👍 ok good to know